CaptureSDK/GiniCaptureSDK/Tests/GiniCaptureSDKTests/ImageMetaInformationManagerTests.swift

```diff
@@ -62,5 +62,42 @@ final class ImageMetaInformationManagerTests: XCTestCase {
         XCTAssert((mutableInformation.getMetaInformation(forKey: key) as? String)?.contains("GiniCaptureVer") == true,
                   "filtered data did not set custom fields")
     }
+    
+    func testSettingDefaultEntryPointToMetaInformation() {
+        guard let mutableInformation = manager.metaInformation?.mutableCopy() as? NSMutableDictionary else {
+            return XCTFail("failed to retrieve mutable meta information from test data")
+        }
+        let value = "button"
+        let key = kCGImagePropertyTIFFMake as String
+        mutableInformation.set(metaInformation: value as AnyObject?, forKey: key)
+        XCTAssert(mutableInformation.getMetaInformation(forKey: key) as? String == value,
+                  "failed to set value for entry point correctly on meta information")
+    }
+    
+    func testSettingFieldEntryPointToMetaInformation() {
+        guard let mutableInformation = manager.metaInformation?.mutableCopy() as? NSMutableDictionary else {
+            return XCTFail("failed to retrieve mutable meta information from test data")
+        }
+        let value = "field"
+        let key = kCGImagePropertyTIFFMake as String
+        mutableInformation.set(metaInformation: value as AnyObject?, forKey: key)
+        XCTAssert(mutableInformation.getMetaInformation(forKey: key) as? String == value,
+                  "failed to set value for entry point correctly on meta information")
+    }
+    
+    func testFieldEntryPointToMetaInformation() {
+        let metaManager = ImageMetaInformationManager(imageData: GiniCaptureTestsHelper.fileData(named: "testFieldEntryPoint", fileExtension: "jpg")!, imageSource: .camera)
+        let metaInformation = metaManager.metaInformation as? NSMutableDictionary
+        let value = "field"
+        let existingUserComment = metaInformation?.getMetaInformation(forKey: kCGImagePropertyExifUserComment as String)
+        let components = existingUserComment?.components(separatedBy: ",")
+        let userCommentComponent = components?.filter({ (component) -> Bool in
+            return component.contains("EntryPoint")
+        })
+        let equasionComponents = userCommentComponent?.last?.components(separatedBy: "=")
+        let entryPoint = equasionComponents?.last
+        XCTAssert(entryPoint == value,
+                  "failed to set value for entry point correctly on meta information")
+    }
```

Comment:
[Request] The first two tests test that you can set and read the "make" exif property (`kCGImagePropertyTIFFMake`) with the values "field" and "button". That's not in line with the test method names and also doesn't exercise the code lines you added. So the first two tests you can delete.

The third test does use the code lines you added and I would like to ask you duplicate it for testing the button entry point, too. You could call it `testButtonEntryPointToMetaInformation`.